frontend/src/bundles/users/components/user-card/components/user-avatar.tsx

```diff
@@ -1,18 +1,19 @@
 import { Circle } from '~/bundles/common/components/components.js';
+import { useAppSelector } from '~/bundles/common/hooks/hooks.js';
 
-type Properties = {
-    username: string;
-};
+import { getInitials } from '../helpers/helpers.js';
 
-const UserAvatar: React.FC<Properties> = ({ username }) => {
+const UserAvatar: React.FC = () => {
+    const { user } = useAppSelector(({ auth }) => auth);
```

Comment:
```suggestion
   const { user } = useAppSelector(({ auth }) => ({
      user: auth.user,
   }));
```
We are using it like this everywhere, so I guess it is better to be consistent with our code style